Jenkinsfile

```diff
@@ -41,18 +41,11 @@ node('java') {
         }
 
         stage ('Run') {
-            def javaArgs = ' -DdefinitionsDir=$PWD/permissions' +
-                        ' -DartifactoryApiTempDir=$PWD/json' +
-                        ' -DartifactoryUserNamesJsonListUrl=https://reports.jenkins.io/artifactory-ldap-users-report.json' +
-                        ' -Djava.util.logging.SimpleFormatter.format="%1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS %4$s: %5$s%6$s%n"' +
-                        ' -jar target/repository-permissions-updater-*-bin/repository-permissions-updater-*.jar'
-
-
             if (dryRun) {
-                sh '${JAVA_HOME}/bin/java -DdryRun=true' + javaArgs
+                sh './update-permissions.sh'
```

Comment:
Didn't bother making sure to reuse the path of the installed `JAVA_HOME`, it seems to still work. But can do if you think this is important.